types/react-onsenui/index.d.ts

```diff
@@ -429,10 +429,10 @@ export class LazyList extends Component<{
     calculateItemHeight(rowIndex: number): any,
 }, any> { }
 
-export class List extends Component<{
+export class List <T> extends Component<{
```

Comment:
We don't usually put a space before the type parameter list.

Reply:
@amcasey I have removed the extra space and added an example.